broker/broker_utils/broker_utils/data_utils.py

```diff
@@ -33,16 +100,43 @@ def alert_dict_to_dataframe(alert_dict: dict, schema_map: dict) -> pd.DataFrame:
     """ Packages an alert into a dataframe.
     Adapted from: https://github.com/ZwickyTransientFacility/ztf-avro-alert/blob/master/notebooks/Filtering_alerts.ipynb
     """
+    if not isinstance(schema_map, dict):
+        raise ValueError("`schema_map` is not a valid dictionary.")
+
     dfc = pd.DataFrame(alert_dict[schema_map['source']], index=[0])
     df_prv = pd.DataFrame(alert_dict[schema_map['prvSources']])
```

Comment:
Super minor thing, but "df_prv" has an underscore while the other two variables "dfc" and "dflc" don't.

Reply:
Thanks, and all those names are pretty bad to begin with. I took the opportunity to rename them all, hopefully it's a bit better.